### PR TITLE
Improvement/improvements to table

### PR DESCRIPTION
**Pull request by ognyan-kostadinov**
**Component**:

**Description**:

**Design**:

**Breaking Changes**:

- [] Breaking Changes

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: src/lib/components/tablev2/SingleSelectableContent.tsx [hunk 1]
```diff
@@ -203,6 +207,16 @@ export function SingleSelectableContent<
...
+      {isLoadingMoreItems && (
+        <Box
+          display="flex"
+          justifyContent="center"
+          marginTop={spacing.r16}
+          marginBottom={spacing.r16}
+        >
+          <Loader size="large" />
+        </Box>
+      )}
```
Comment (JBWatenbergScality) [inline src/lib/components/tablev2/SingleSelectableContent.tsx:219]: This change should also be done in the Multiselectable table

File: src/lib/components/constrainedtext/Constrainedtext.component.tsx [hunk 2]
```diff
@@ -22,7 +22,6 @@ const ConstrainedTextContainer = styled.div`
   ${(props) =>
     props.lineClamp > 1
       ? `
-  display: -webkit-box;
```
Comment (JBWatenbergScality) [inline src/lib/components/constrainedtext/Constrainedtext.component.tsx:25]: Can you describe the use case where the ellipsis was not displayed ? According to mdn docs we can't use webkit-line-clamp without display: -webkit-box or -webkit-inline-box (https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp):

> The -webkit-line-clamp CSS property allows limiting of the contents of a [block](https://developer.mozilla.org/en-US/docs/Glossary/Block) to the specified number of lines.

> It only works in combination with the [display](https://developer.mozilla.org/en-US/docs/Web/CSS/display) property set to -webkit-box or -webkit-inline-box and the [-webkit-box-orient](https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient) property set to vertical.

So bottomline I'm afraid that this change breaks the line-clamp "feature" of the component.

Comment (ognyan-kostadinov) [reply src/lib/components/constrainedtext/Constrainedtext.component.tsx:25]: The ellipsis was not displayed on any column of Groups or Users table (checked both on Firefox and Chrome). I saw the comment and the documentation, but I think it's outdated

Comment (ognyan-kostadinov) [reply src/lib/components/constrainedtext/Constrainedtext.component.tsx:25]: Here are some "before and after" screenshots: 
before-core-ui-ellipsis-fix

after-core-ui-ellipsis-fix

Comment (ognyan-kostadinov) [reply src/lib/components/constrainedtext/Constrainedtext.component.tsx:25]: Okay, I understood the nature of the problem. When `display: -webkit-box;` and `-webkit-line-clamp` are used together, ellipsis are placed only when the content exceeds the given number of lines (in our case 2). When the content is less than the given lines, the behavior is similar to `overflow: hidden` (first screenshot)

Comment (ognyan-kostadinov) [reply src/lib/components/constrainedtext/Constrainedtext.component.tsx:25]: I'll revert this change. It's less than ideal for shorter text, but we avoid breaking the table for longer one

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**